search-scores-and-weights.md

```diff
@@ -0,0 +1,40 @@
+Our search system uses scoring to assign values to each document based on
+different parameters. The interplay between clustering and scoring helps
+optimize the search process, ensuring that the system considers both the content
+and context to deliver more accurate and relevant results for the user.
+
+This task becomes challenging when dealing with a large number of documents,
+creating the need for optimization strategies. A good approach is the
+utilization of clustering techniques, which categorize documents based on their
+topics, facilitating a more streamlined and efficient search process. We also
+use scoring, which involves the assignment of numerical values and weights to
+documents based on various parameters, influencing their ranking in the search
+results. Here is our different scores, each serving a unique purpose:
+
+1. **Recency**: This score considers the temporal aspect of documents,
+   prioritizing recently added or updated content. A document's recency is
+   crucial in reflecting the latest information available to users.
+
+1. **Traffic**: The frequency with which users consult a document influences its
+   score. Popular or frequently accessed documents are given higher scores,
+   indicating their relevance and importance to users. Warning: The home page is
+   rated really high since it's where every user land at first.
+
+1. **Current**: This score determines whether a document is currently accessible
+   (current=1) or if it has been archived (current=0). It helps users
+   distinguish between active and inactive content.
+
+1. **Typicality**: This score assesses whether a document is a typical
```

Comment:
that's not what it is doing now. please go back to the source SQL

Reply:
done